app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter(post)
+    post.comments_counter = post.comments_counter.to_i + 1
+    post.save!
+  end
```

Comment:
- [ Required ] I think it would be nice if you use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new `Comment` has been saved to the database, then make the update_comments_counter! method private. This will ensure that the comments counter is incremented when a new comment has been saved to the database, please the suggestion below for an idea of how to achieve this.

## Example code
```ruby
class Comment < ApplicationRecord
  belongs_to :user
  belongs_to :post

  after_save :update_comments_counter

  private

  def update_comments_counter
    post.update(comments_counter: post.comments.count)
  end
end
```